examples/bigcommerce/lib/bigcommerce/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { getProduct, getProducts } from './bigcommerce'
+
+export const getProductSlugPaths = async (): Promise<string[]> => {
+  const products = await getProducts()
+
+  return products?.map((edge: any) => edge.node.entityId)?.map((entityId: any) => entityId) ?? []
```

Comment:
Are we planning on leaving these types as `any`? We could manually define some types or use some code generation to leverage TypeScript and GraphQL